packages/xo-server/src/api/token.mjs

```diff
@@ -25,35 +25,17 @@ create.params = {
 
 // -------------------------------------------------------------------
 
-async function delete_({ token: id }) {
-  await this.deleteAuthenticationToken(id)
+async function delete_({ pattern, tokens }) {
+  await this.deleteAuthenticationTokens({ filter: pattern ?? { id: { __or: tokens } } })
```

Comment:
`deleteAuthenticationTokens` has not been updated to take `apiContext` into account, your change allow all users to delete all tokens.

Reply:
`deleteAuthenticationTokens` is a wrapper that do a `Promise.all` of [deleteAuthenticationtoken](https://github.com/vatesfr/xen-orchestra/blob/31ed477b9661102b670213f4b5e61601838f110f/packages/xo-server/src/xo-mixins/authentication.mjs#L216)

Reply:
I see, thanks :slightly_smiling_face: